PROCESSORS.md

```diff
@@ -1358,7 +1358,7 @@ In the list below, the names of required properties appear in bold. Any other pr
 | send-message-body                               | true                     | true<br/>false                                                                       | DEPRECATED. Only kept for backwards compatibility, no functionality is included.                                                                                                                                                                                                                                |
 | Send Message Body                               | true                     | true<br/>false                                                                       | If true, sends the HTTP message body on POST/PUT/PATCH requests (default). If false, suppresses the message body and content-type header for these requests.                                                                                                                                                    |
 | Use Chunked Encoding                            | false                    | true<br/>false                                                                       | When POST'ing, PUT'ing or PATCH'ing content set this property to true in order to not pass the 'Content-length' header and instead send 'Transfer-Encoding' with a value of 'chunked'. This will enable the data transfer mechanism which was introduced in HTTP 1.1 to pass data of unknown lengths in chunks. |
-| Disable Peer Verification                       | false                    | true<br/>false                                                                       | Disables peer verification for the SSL session                                                                                                                                                                                                                                                                  |
+| Disable Peer Verification                       | false                    | true<br/>false                                                                       | DEPRECATED. Only kept for backwards compatibility, no functionality is included.                                                                                                                                                                                                                                |
```

Comment:
Deprecated already communicates that this is only kept for backwards compatibility. I suggest changing the sentence to describe how the value is used, and what's the actual behavior. Users reading this description are most likely interested in these details.
```suggestion
| Disable Peer Verification                       | false                    | true<br/>false                                                                       | DEPRECATED. The value is ignored, peer and host verification are always performed when using SSL/TLS.                                                                                                                                                                                                           |
```

Reply:
Makes sense, updated in 55a700da1079435db2a352da9e672c6fa5ce2501